### PR TITLE
Issue #15 Feature pull request template

### PR DESCRIPTION
**Pull request by IgnatiusFlo**
Hello,
I hope this is an adequate pull requst template.
😸

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Provide a simple description of what the PR achieves. 
+
+## Pull Request Details
+
+Please ensure you PR fulfills the following criteria:
+
+- [ ] Does your PR solve an issue created by a member of staff or event handler?
+- [ ] Have you joined the Discord community? 
+- [ ] Were you changes made in the Pipenv environment?
```
Comment (jos-b) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: `you` should be `your`

**Comment by IgnatiusFlo** [comment]
Thanks for providing some low hanging fruit for is noobs. 🍒

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+Provide a simple description of what the PR achieves. 
+
+## Pull Request Details
+
+Please ensure you PR fulfills the following criteria:
+
+- [ ] Does your PR solve an issue created by a member of staff or event handler?
```
Comment (jos-b) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I'm not sure whether we need this one, while we do provide a list of issues to fix we welcome other PRs as long as they achieve something meaningful.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Pull Request
+about: A simple pull request template.
+
```
Comment (jos-b) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Maybe we could add another column for the issue ID solved if applicable?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Provide a simple description of what the PR achieves. 
+
+## Pull Request Details
+
+Please ensure you PR fulfills the following criteria:
+
+- [ ] Does your PR solve an issue created by a member of staff or event handler?
+- [ ] Have you joined the Discord community? 
+- [ ] Were your changes made in the Pipenv environment?
+- [ ] Is your commit message clear and references the issue?
```
Comment (jos-b) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Not sure this is required. PRs can have multiple commits & if some are lousily worded then I'm sure we can get by using the body of the PR and the changes made.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Provide a simple description of what the PR achieves. 
+
+## Pull Request Details
+
+Please ensure you PR fulfills the following criteria:
+
+- [ ] Does your PR solve an issue created by a member of staff or event handler?
+- [ ] Have you joined the Discord community? 
+- [ ] Were your changes made in the Pipenv environment?
```
Comment (jos-b) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: `A Pipenv environment` feels better than `the Pipenv environment` to me

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+name: Pull Request
+about: A simple pull request template.
+
+---
+
+Provide a simple description of what the PR achieves. 
+
+## Pull Request Details
+
+Please ensure you PR fulfills the following criteria:
```
Comment (jos-b) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Should be `your PR`

**Comment by sco1** [comment]
I would probably include a line for ensuring that `flake8` passes (`pipenv run lint`) before submitting the PR.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 2 above)
Comment (IgnatiusFlo) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Sure, I just took the points from the contribution guide.

**Comment by IgnatiusFlo** [comment]
Okay. I've made those changes. Any other thoughts on what you'd like included? 

I'm not sure of the appropriate grammar involved in discussing flake8, but I included the command.

**Comment by IgnatiusFlo** [comment]
Thanks!

**Pull request merged**